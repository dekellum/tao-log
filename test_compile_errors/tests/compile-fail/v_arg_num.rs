//! Tests of *v argument count

extern crate tao_log;
use tao_log::*;

fn zero_arguments() {
    infov!(); //~ ERROR unexpected end of macro invocation
}

fn four_arguments() {
    infov!("prefix", "{}", true, 4); //~ ERROR expected identifier, found `,`
}
