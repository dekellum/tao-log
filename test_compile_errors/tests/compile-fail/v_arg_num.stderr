error: unexpected end of macro invocation
 --> $DIR/v_arg_num.rs:7:5
  |
7 |     infov!();
  |     ^^^^^^^^^ missing tokens in macro arguments

error: no rules expected the token `4`
  --> $DIR/v_arg_num.rs:11:34
   |
11 |     infov!("prefix", "{}", true, 4);
   |                                  ^ no rules expected this token in macro call

error[E0601]: `main` function not found in crate `$CRATE`
  |
  = note: consider adding a `main` function to `$DIR/tests/compile-fail/v_arg_num.rs`

For more information about this error, try `rustc --explain E0601`.
